modules/jupyter-widget/src/playground/playground.js

```diff
@@ -96,7 +93,8 @@ function sendEventViaTransport(transport, event, data) {
   }
   // TODO Remove circular references without converting to a string
   const deckEvent = JSON.parse(JSON.stringify({event, data}, filterJsonValue));
-  transport.jupyterModel.send(deckEvent);
+
+  transport.sendJSONMessage('deck-event', deckEvent);
```

Comment:
@ajduberstein Note that this adds an additional level of nesting that needs to be matched on Python side.
- A transport message has `{type, data}`
- A deck.gl event will be

```js
{
  type: 'deck-event',
  data: {
    type: 'click',
    ...
  }
}
```

Reply:
We could avoid this nesting but I recommend a naming convention to separate deck events from other back-channel events. Le me update that.